### PR TITLE
Added FullTimeEquivalency value to StudentSchoolAssociations

### PR DESCRIPTION
**Pull request by markramonDL**
added <FullTimeEquivalency>1.00</FullTimeEquivalency> to all StudentSchoolAssociation records.

### CODE REVIEW

File: Samples/Sample XML/StudentEnrollment.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		<EntryDate>2021-08-23</EntryDate>
 		<EntryGradeLevel>uri://ed-fi.org/GradeLevelDescriptor#Fifth grade</EntryGradeLevel>
 		<GraduationPlanReference ref="GPLN_Minimum-GBES"/>
-		<CalendarReference>
+ 		<CalendarReference>
```
Comment (blmeyers) [inline Samples/Sample XML/StudentEnrollment.xml:17]:
```suggestion
		<CalendarReference>
```

File: Samples/Sample XML/StudentEnrollment.xml [hunk 2]
```diff
@@ -2005,7 +2137,7 @@
 		<EntryDate>2021-08-23</EntryDate>
 		<EntryGradeLevel>uri://ed-fi.org/GradeLevelDescriptor#First grade</EntryGradeLevel>
 		<GraduationPlanReference ref="GPLN_Minimum-GBES"/>
-		<FullTimeEquivalency>0.75</FullTimeEquivalency>
+		<FullTimeEquivalency>1.00</FullTimeEquivalency>
```
Comment (blmeyers) [inline Samples/Sample XML/StudentEnrollment.xml:2140]: These existing 0.75 values probably shouldn't be defaulted to 1.00.

File: Samples/Sample XML/StudentEnrollment.xml [hunk 3]
```diff
@@ -2367,7 +2521,7 @@
...
```
Comment (blmeyers) [inline Samples/Sample XML/StudentEnrollment.xml:2524]: These existing 0.75 values probably shouldn't be defaulted to 1.00.

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: Samples/Sample XML/StudentEnrollment.xml (hunk 1 above)
Comment (markramonDL) [reply Samples/Sample XML/StudentEnrollment.xml:17]: This was just a mistake and I restored the <CalendarReference> and file validates.

File: Samples/Sample XML/StudentEnrollment.xml (hunk 2 above)
Comment (markramonDL) [reply Samples/Sample XML/StudentEnrollment.xml]: restored original value.

File: Samples/Sample XML/StudentEnrollment.xml (hunk 3 above)
Comment (markramonDL) [reply Samples/Sample XML/StudentEnrollment.xml]: restored original value.

File: Samples/Sample XML/StudentEnrollment.xml (hunk 1 above)
Comment (blmeyers) [reply Samples/Sample XML/StudentEnrollment.xml:17]: This is still showing as extra white space being added in GitHub.

**Review by blmeyers — APPROVED** [review]
@ggaronzik @ejansson Looks good from my end, just introducing FullTimeEquivalency data for all the students, leaving it with original values for the ones who already had it.

**Pull request merged**